### DISCUSSION TITLE
psxita update

### DISCUSSION THREAD

**Discussion by danoddo88** [hippie68/psxitarch-how-to#11] [Q&A]
Hello!  thanks for your updated psxita linux, im traying to update it but it continously crashes, im ignoring at pacman.conf this packages: mesa-git lib32-mesa-git libdrm-git lib32-libdrm-git xf86-video-amdgpu-git... do you know if im missing any other packages? so if i use sudo pacman -Syu does not crash the jwm (im new with arch)? thanks in advance

**Comment by hippie68** [comment]
I am not aware of crashes or having to ignore packages. Are you trying to update the original psxitarch release or the Aug 22, 2021 version from the release section? It should be possible to close jwm and update from within the command line, too.

**Reply by danoddo88** [reply]
Hello, thank you very much for answering me. im trying to update your psxitarch (aug 22 - 2021) from this year. when i use sudo pacman -Syu there is a conflict with xorg-server and Xf86-video-ati-git, i choose to delete Xf86-video-ati-git and install it after it finish the full update but it doesnt work, when i reboot the desk does not appear, only a black screen
![image](https://user-images.githubusercontent.com/95310310/146989932-ea34175e-6b6a-4c2a-911c-0abf09b27dab.png)
